omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml

```diff
@@ -301,6 +301,16 @@
             });
 
             jq(document).ready(function () {
+                /* Prompt for malnutrition using Muac and BMI */
+                function BMI(ht,wt){
+                   var htInMetres = ht/100;
+                 return  wt/(htInMetres * htInMetres);
+                }
+
+               if((jq("#lastMuac").html() &lt; 19) || (BMI(jq("#lastHeight").html(),jq("#lastWeight").html()) &lt; 18.5) ){
```

Comment:
@reagan-meant use getValue() instead of the jquery option 

https://wiki.openmrs.org/display/docs/HTML+Form+Entry+JavaScript+Reference#HTMLFormEntryJavaScriptReference-getValue(idAndProperty)

Reply:
getValue() returns null because there are no values set when using look up tags; instead we get the html as seen [in these references](https://github.com/METS-Programme/openmrs-module-ugandaemr-iss/blob/master/omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-EncounterPage.xml#L370-L375)
and so I have mantained this pattern